packages/roosterjs-editor-dom/lib/utils/applyTableFormat.ts

```diff
@@ -286,10 +310,19 @@ function setHeaderRowFormat(cells: VCell[][], format: TableFormat) {
     }
     cells[0]?.forEach(cell => {
         if (cell.td && format.headerRowColor) {
-            cell.td.style.backgroundColor = format.headerRowColor;
-            cell.td.style.borderRightColor = format.headerRowColor;
-            cell.td.style.borderLeftColor = format.headerRowColor;
-            cell.td.style.borderTopColor = format.headerRowColor;
+            setColor(cell.td, format.headerRowColor, true, isDarkMode);
+            cell.td.style.borderRightColor = setHeaderRowBorderColors(
```

Comment:
actually our color transformation code won't transform border color. So let's always use light mode color here. Otherwise it is another bug

Reply:
If we do not change the border color for the header row, we can see a difference between the background aborder color. 
![Screen Shot 2022-02-21 at 16 54 19](https://user-images.githubusercontent.com/87443959/155218881-7b3bfdd4-bba7-4c47-a60f-2e77a62c2b0a.png)

Reply:
Do we use always light mode for header row background color?

Reply:
Background color need to be transformed to dark color. And in theory, we need to do it for border color too. But it is not down in transformColor